### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by gkapoor13**
Please review

### CODE REVIEW

File: Model/src/main/java/com/aires/devops/model/ModelController.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
 public class ModelController {
     private String outputText; 
         
+    private String outputText1; 
+
```
Comment (vrssun84) [inline Model/src/main/java/com/aires/devops/model/ModelController.java]: Please add setter and getters.

File: ViewController/public_html/index.jsf [hunk 2]
```diff
@@ -4,7 +4,10 @@
     <af:document title="index.jsf" id="d1">
         <af:form id="f1">
         <af:panelGroupLayout id="pgl1" layout="horizontal" halign="center">
-            <h1 class="starter-templatesdfsd"> Welcome to Aires DevOps</h1>
+            <h1 class="starter-templatesdfsd"> Welcome to aires DevOps</h1>
```
Comment (vrssun84) [inline ViewController/public_html/index.jsf]: This looks good.

File: Model/src/main/java/com/aires/devops/model/ModelController.java (hunk 1 above)
Comment (gkapoor13) [reply Model/src/main/java/com/aires/devops/model/ModelController.java]: Ok will make a request

Comment (gkapoor13) [reply Model/src/main/java/com/aires/devops/model/ModelController.java]: Added the getters and setters

Comment (vrssun84) [reply Model/src/main/java/com/aires/devops/model/ModelController.java]: Looks Good.

**Comment by vrssun84** [comment]
Closing the Pull the request

**Pull request closed without merging**